### ISSUE TITLE
SdlDisconnectedReason NullPointerException

### ISSUE THREAD

**Issue by MikaylaRay44** [smartdevicelink/sdl_android#202]
> _com/smartdevicelink/rpc/enums/SdlDisconnectedReason_

`NullPointerException` is thrown by the `convertAppInterfaceUnregisteredReason` method in the [SdlDisconnectedReason](https://github.com/smartdevicelink/sdl_android/blob/develop/sdl_android_lib/src/com/smartdevicelink/proxy/rpc/enums/SdlDisconnectedReason.java) unit test class.

```java
assertEquals(Test.MATCH, 
    SdlDisconnectedReason.DEFAULT, 
    SdlDisconnectedReason.convertAppInterfaceUnregisteredReason(null));
```

Solution : The method should check that the parameter is not null before using it in a switch statement.

**Comment by mrapitis** [comment]
Both the master and dev branches should already have this resolved. The fix was implemented in #119

**Comment by MikaylaRay44** [comment]
Thank you, yes this change isn't currently reflected in the unit_test branch.
